scripts/plot_scalar_results.py

```diff
@@ -702,6 +702,74 @@ def subplot_flow_out_multi_carrier(carriers):
         except Exception as e:  # noqa 722
             logger.warning(f"Could not plot {output_path_plot}: {e}.")
 
+    def plot_demands_stacked_carriers(carriers):
+        carriers.append("ch4")
```

Comment:
Better not do this inside the function.

Reply:
Done with commit https://github.com/rl-institut/oemof-B3/pull/228/commits/be82a01bee76a5f22b09df24a4951888930b37f7.